app/script/view_model/content/MessageListViewModel.js

```diff
@@ -261,6 +268,7 @@ z.viewModel.content.MessageListViewModel = class MessageListViewModel {
         }
 
         $('.conversation').css({opacity: 1});
+        this.capture_scrolling_event = true;
```

Comment:
This is the actual fix, enabling scroll handling **after** the programmatic scroll have occurred....

On a side note, this will probably annoy us again in the future, so we might need to think of a better way to do that.
A different API to explicitly scroll into the messages without triggering listeners could be a solution.... Thinking in progress

Reply:
Wow, I am actually not even sure this will fix the issue.
As we throttle the handling of scroll events, we might end up handling a past event with the boolean set to `true`....